components/builder-protocol/src/originsrv.rs

```diff
@@ -980,6 +980,38 @@ impl Serialize for OriginProject {
     }
 }
 
+// impl Routable for ProjectStateSet {
```

Comment:
Did you mean to add this?

Reply:
No, it was left over from a really old branch. Thanks!